README.md

```diff
@@ -88,6 +84,7 @@ There are many projects under the stewardship of the .NET Foundation. The [full
 - [Open Live Writer](http://openlivewriter.org)
 - [Open Office SDK](https://github.com/officedev/open-xml-sdk)
 - [Reactive Extensions for .NET](https://github.com/reactive-extensions/rx.net)
+- [SourceLink](https://github.com/dotnet/SourceLink)
```

Comment:
`[Source Link](https://github.com/dotnet/sourcelink)` is probably better. The Visual Studio team added the space and everyone else adjusted to add it too instead of asking them to correct it. The repo URL is lowercase.

Reply:
@ctaggart thanks for that suggestion.  I have updated this PR with that change.